rasa_nlu/utils/__init__.py

```diff
@@ -254,6 +258,12 @@ def read_yaml(content):
     yaml_parser.version = "1.2"
```

Comment:
Can we get a docstring for this function so that it is a bit clearer what the regex function is doing? 👮🏻‍♂️